### PR TITLE
Allow Panacea to load all DLLs from a subfolder.

### PR DESCRIPTION
**Pull request by TopazTK**
Please test, this is hacky.

### CODE REVIEW

File: OpenKh.Research.Panacea/OpenKh.Research.Panacea.vcxproj [hunk 1]
```diff
@@ -87,6 +89,9 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <EnableUAC>false</EnableUAC>
       <AdditionalDependencies>Shlwapi.lib;Dependencies\bass.lib;Dependencies\bass_vgmstream.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalLibraryDirectories>
+      </AdditionalLibraryDirectories>
```
Comment (shananas) [inline OpenKh.Research.Panacea/OpenKh.Research.Panacea.vcxproj:93]: can confirm these two lines above are not needed.

Comment (TopazTK) [reply OpenKh.Research.Panacea/OpenKh.Research.Panacea.vcxproj:93]: lmfao let them stay, C++ is weird.

Comment (shananas) [reply OpenKh.Research.Panacea/OpenKh.Research.Panacea.vcxproj:93]: but it'll bother me.

Comment (TopazTK) [reply OpenKh.Research.Panacea/OpenKh.Research.Panacea.vcxproj:93]: Alright then. You are responsible if it breaks :I

Comment (shananas) [reply OpenKh.Research.Panacea/OpenKh.Research.Panacea.vcxproj]: oh. it wont I tested it before saying it was unneeded.

**Review by shananas — APPROVED** [review]

**Comment by TopazTK** [comment]
I am merging after 1 more review. Just in case.

**Review by Vladabdf — APPROVED** [review]
Just merge the stuff

**Comment by MainMemory** [comment]
cool thanks

**Pull request merged**